commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/TableModel.kt

```diff
@@ -68,11 +69,16 @@ abstract class Row<T>(val original: T) {
 
     override fun hashCode() = Objects.hash(original)
 
-    fun headers() = attributes.map { Header(it.name, it.nameAttrs.map(AttributeDetail::name)) }
+    fun headers() = attributes.map {
+        Header(it.name, it.nameAttrs.map(AttributeDetail::name), it.valueAttrs.map(AttributeDetail::name))
```

Comment:
I am wondering if is worth it to create attribute extension properties to simplify code here. something like 

```
    val nameDetailsNames: List<String>
        get() = nameAttrs.map(AttributeDetail::name)
```

so it becomes

` Header(it.name, it.nameDetailsNames, it.valueDetailsNames`

Reply:
Done